roles/config-wireguard/config/fra.yml

```diff
@@ -332,3 +332,10 @@ wg_peers:
   wg_pubkey: 7gbJe7eeFxHdZnHnlBA1H33HHLqnmGjgSWwsOLimtlM=
   peer_v4: 172.20.193.163
   peer_v6: fd4e:d0:d38d::3
+
+- name: dn42-divinet
+  port: 21409
+  remote: nym.dn42.join.yt:23002
```

Comment:
It looks like this connection has significantly higher ping (~30ms) than the other one (~1s), plus this endpoint is closer to my AMS node. In these cases I recommend only keeping the lower-latency peer config.